apps/emqx/src/emqx_listeners.erl

```diff
@@ -304,9 +304,9 @@ update_listener(Type, Name, Conf = #{enable := true}, #{enable := false}) ->
 update_listener(Type, Name, #{enable := false}, Conf = #{enable := true}) ->
     start_listener(Type, Name, Conf);
 update_listener(Type, Name, OldConf, NewConf) ->
+    ok = emqx_limiter:update_listener_limiters(listener_id(Type, Name), NewConf),
```

Comment:
will the bucket be reset even if limiter config in `OldConf` and `NewConf` are identical?

Reply:
nice. please add.